### PR TITLE
APM-234 & APM-263

### PR DESCRIPTION
**Pull request by MollieRick**

### CODE REVIEW

File: source/reference/v2/methods-api/get-method.rst [hunk 1]
```diff
@@ -39,6 +39,13 @@ Replace ``id`` in the endpoint URL by the methods's ID. For example: ``https://a
        ``ca_ES`` ``pt_PT`` ``it_IT`` ``nb_NO`` ``sv_SE`` ``fi_FI`` ``da_DK`` ``is_IS`` ``hu_HU`` ``pl_PL`` ``lv_LV``
        ``lt_LT``
 
+   * - ``currency``
+
+       .. type:: string
+          :required: false
+
+     - The currency to receiving the ``minimumAmount`` and ``maximumAmount`` in.
```
[the remark this answers is not in the document]
Comment (MollieRick) [reply source/reference/v2/methods-api/get-method.rst]: Will throw a BadRequestException

**Comment by sandervanhooft** [comment]
A reference to this endpoint would be welcome here as well: https://help.mollie.com/hc/en-us/articles/115000667365-What-are-the-minimum-and-maximum-amounts-per-payment-method .

File: source/reference/v2/methods-api/get-method.rst (hunk 1 above)
Comment (willemstuursma) [inline source/reference/v2/methods-api/get-method.rst]: If the currency is not supported for the payment method...

**Comment by MollieRick** [comment]
Problems at Travis. Builds succeeded at local machine.

**Pull request merged**